website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -892,6 +894,61 @@ curl \
 }
 ```
 
+## Show data retention policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`GET /organizations/:organization_name/relationships/data-retention-policy`
+
+| Parameter            | Description                                                         |
+| ---------------------| --------------------------------------------------------------------|
+| `:organization_name` | The name of the organization to show the data retention policy for. |
+
+This endpoint shows the data retention policy set explicitly on the organization.
+
+When no data retention policy is set for the organization, the endpoint returns the default policy configured for the Terraform Enterprise installation. Read more about [organization data retention policies](/terraform/enterprise/users-teams-organizations/organizations#data-retention-policies).
+
+For additional information, refer to [Data Retention Policy Types](/terraform/enterprise/api-docs/data-retention-policies#data-retention-policy-types).
```

Comment:
```suggestion
For additional information, refer to [Data Retention Policy Types](/terraform/enterprise/api-docs/data-retention-policies#data-retention-policy-types) in the Terraform Enterprise documentation.
```
Just to make it clear that the link takes readers to a different set. Please make this change for other links from TFC to TFE.